### DISCUSSION TITLE
LaTeX support in projects and Kanban boards

### DISCUSSION THREAD

**Discussion by dpo** [community/community#18300] [Projects and Issues]
Hi! I'm thrilled that GitHub supports Mathjax syntax to render LaTeX expressions. However, I just tried to insert equations in a Kanban board (in the projects of one of my repos), and it doesn't look like equations are rendered there. Am I missing something? If not, it would be fantastic to add LaTeX support there!

Thanks!

**Comment by kocierik** [comment]
Github only uses _MathJax_ at the moment...

**Reply by dpo** [reply]
I understand that. It doesn't seem to work for me in project boards though.
